code/modules/mob/living/carbon/human/human_defense.dm

```diff
@@ -287,7 +287,16 @@
 			log_combat(user, src, "tackled")
 			visible_message(span_danger("[user] tackles [src] down!"), \
 							span_userdanger("[user] tackles you down!"), span_hear("You hear aggressive shuffling followed by a loud thud!"), null, user)
-			to_chat(user, span_danger("You tackle [src] down!"))
+			to_chat(user, span_danger("You tackle [src] down!"))*/
+		playsound(loc, 'sound/weapons/pierce.ogg', 25, TRUE, -1) //monkestation edit start
+		Knockdown(2 SECONDS)
+		log_combat(user, src, "tackled")
+		visible_message(span_danger("[user] tackles [src] down!"), \
+						span_userdanger("[user] tackles you down!"), span_hear("You hear aggressive shuffling followed by a loud thud!"), null, user)
+		to_chat(user, span_danger("You tackle [src] down!"))
+		var/obj/item/bodypart/affecting = get_bodypart(get_random_valid_zone(user.zone_selected))
+		var/armor_block = run_armor_check(affecting, MELEE,"","",10)
+		apply_damage(30, STAMINA, affecting, armor_block) //monkestation edit end
```

Comment:
stamina damage application is different now, please use user.stamina.adjust(-30) instead